script/test

```diff
@@ -8,4 +8,4 @@ if [ $CI ] ; then
   npm ci
 fi
 
-npm run test
+npm run test "$@"
```

Comment:
Pass arguments to the npm script - useful for updating tests as we can do `./script/test -- -u`.

Reply:
TIL! 🙏

Reply:
😄 At second look, I'm not too keen on this, `./script/test --update` is a more obvious interface, will update.

Reply:
Slightly better now, we can now pass any [Jest flag](https://jestjs.io/docs/en/cli), for example `./script/test -u`.